iati/default.py

```diff
@@ -200,26 +206,28 @@ def ruleset(version=None):
         iati.Ruleset: The default Ruleset for the specified version of the Standard.
 
     """
+    version = _specific_version_for(version)
+
     path = iati.resources.get_ruleset_path(iati.resources.FILE_RULESET_STANDARD_NAME, version)
     ruleset_str = iati.utilities.load_as_string(path)
 
     return iati.Ruleset(ruleset_str)
 
 
-def ruleset_schema(version=None):
+def ruleset_schema():
     """Return the Ruleset schema for the specified version of the Standard.
 
-    Args:
-        version (str): The version of the Standard to return the Ruleset for. Defaults to None. This means that a version-independent Ruleset schema is returned.
-
     Raises:
         ValueError: When a specified version is not a valid version of the IATI Standard.
 
     Returns:
         dict: A dictionary representing the Ruleset schema for the specified version of the Standard.
 
+    Todo:
+        Determine whether a version should be provided.
```

Comment:
Suggest add to this - `Determine whether a version should be provided. This is worth considering if the content of the IATI Ruleset Schema varies between versions`.

Reply:
6f85bf9c4ca588a34baa190e3e2b416f7c81b09a